script.js

```diff
@@ -18,10 +20,104 @@ $(document).ready(function() {
     error: function() {
       alert("there has been an error...")
     }
-  })
-});
+  });
+
+
 
 function handleResponse(response) {
-  console.log(response);
-  // add stuff here!
-}
\ No newline at end of file
+  imgCount = response.data.length;
+  for (var i = 0; i <= response.data.length-1; i++) {
+    $(".instaimages").append("<img class=pics src='"+response.data[i].images.standard_resolution.url+"'/>");
+    $(".instaimages").append(response.data[i].caption.text);
+  } 
+  getEgoResponse();
+  averageNumberLikes(response);
+  avgCaptionLength(response);
+  visibilityThirst(response);
+}
+
+// this styling code does not work, I don't know why
+$("img.pics").css({"max-height": "500px", "max-width": "480px", "border-radius": "4px"});
+
+
+//Stage 3: Stats on Stats - Ego Score
+var EGO_API_DOMAIN = "https://api.instagram.com/v1/users/self/media/liked?access_token=2199392036.37c56ef.08b399627ebf4e26bfb48b7966b5ea53" 
+
+
+  function getEgoResponse() {
```

Comment:
Instead of making a separate .ajax call, you can use the same response data for the stats section